src/index.ts

```diff
@@ -65,20 +65,21 @@ app.post("/run", async (c) => {
     return c.html("<h2>All fields are required!</h2><a href='/'>Back</a>");
   }
 
-  const response = await fetch(`http://localhost:8080/run`, {
-    method: 'POST',
+  const container = getContainer(c.env.MAPI_CONTAINER);
+  return await container.fetch(c.req.raw, {
+    method: "POST",
     headers: {
-      'Content-Type': 'application/x-www-form-urlencoded',
+      "Content-Type": "application/x-www-form-urlencoded",
     },
-    body: new URLSearchParams(
-      Object.fromEntries(
-        Object.entries(body).map(([k, v]) => [k, typeof v === "string" ? v : ""])
-      )
-    ).toString(),
+    body: new URLSearchParams({
+      workspace,
```

Comment:
of course, there are plenty more params we can put here, but again just for poc this is fine